internal/transform/ledger.go

```diff
@@ -0,0 +1,127 @@
+package transform
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/stellar/go/xdr"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+//ConvertLedger converts a ledger from the history archive ingestion system into a form suitable for BigQuery
+func ConvertLedger(inputLedgerMeta xdr.LedgerCloseMeta) (LedgerOutput, error) {
+	ledger, ok := inputLedgerMeta.GetV0()
+	if !ok {
+		return LedgerOutput{}, errors.New("Could not access the version 0 information for the provided ledger")
+	}
+	ledgerHeaderHistory := ledger.LedgerHeader
+	ledgerHeader := ledgerHeaderHistory.Header
+
+	outputSequence := int32(ledgerHeader.LedgerSeq)
+	if outputSequence < 0 {
+		return LedgerOutput{}, errors.New("The sequence is a negative value")
```

Comment:
The sequence is somewhat vague. Maybe "ledger sequence %d is negative"? It would be helpful if there was some other metadata in that error to identify the problematic ledger as well, maybe a timestamp?